backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/MetadataUtils.java

```diff
@@ -308,6 +309,25 @@ protected static void alterTableName(DSLContext jooq, TableMetadata table, Strin
         .execute();
   }
 
+  protected static List<User> loadUsers(DSLContext jooq, int limit, int offset) {
+    List<User> users = new ArrayList<>();
+    for (Object username :
+        jooq.select(USER_NAME)
+            .from(USERS_METADATA)
+            .orderBy(USER_NAME)
+            .limit(limit)
+            .offset(offset)
+            .fetch()
+            .getValues(USER_NAME)) {
+      users.add(new User((String) username));
+    }
```

Comment:
i find these types of loops very hard to read ( and debug) , why not split the fetching and the loop into two statements , of map the result of the fetch